controllers/loginController.ts

```diff
@@ -179,4 +179,6 @@ export default {
   login: asyncHelper(login),
   sendMail: asyncHelper(sendMail),
   deleteToken,
+  generateLoginUrl,
+  generateToken,
```

Comment:
It seems that these methods were exported for test purpose, they are never called.
We may refactor this => put them in a service. If not done here (already too big PR), create a task to do it later.